guides/plugins/apps/custom-data/custom-entities.md

```diff
@@ -40,6 +40,43 @@ Additionally, to the repository you can also access your custom entities via [Ad
 POST /api/search/custom-entity-blog
 ```
 
+## Custom Fields
```

Comment:
Shouldn't the heading be Custom Entities instead?

Reply:
It's about using custom entities to custom fields, because you can create an custom field of "entity" type and then assign to it an entity, here we are adding support to assign custom entities as the entity type. I am open to suggestions though, maybe "Custom entities as custom fields" ?

Reply:
Okay got it 👍 
How about : "Using Custom Entities to Add Custom Fields" or "Add Custom Fields to Custom Entities"?

Reply:
I used "Using Custom Entities with Custom Fields" I hope that is okay :)